spec/signalfx-transform-helper-spec.js

```diff
@@ -1,5 +1,4 @@
 const signalfxHelper = require('../signalfx-transform-helper');
```

Comment:
could you please add a section to readme explaining how to run unit test and/or setup intellij to run them? can be very brief but since this is not obvious to everyone, let's fix it :)

Reply:
added